File: ts/project/.sdk/tm/lua/utility/result_body.lua
-- ProjectName SDK utility: result_body

local function result_body_util(ctx)
  local response = ctx.response
  local result = ctx.result

  if result ~= nil then
    if response ~= nil and response.json_func ~= nil and response.body ~= nil then
      local json_data = response.json_func()
      result.body = json_data
    end
  end

  return result
end

return result_body_util

File: ts/project/.sdk/tm/py/utility/result_body.py
# ProjectName SDK utility: result_body


def result_body_util(ctx):
    response = ctx.response
    result = ctx.result

    if result is not None:
        if response is not None and response.json_func is not None and response.body is not None:
            json_data = response.json_func()
            result.body = json_data

    return result
